fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -1274,6 +1278,18 @@ private void checkPasswordAndLoadPriv(String user, String passwd, String db, Str
         }
     }
 
+    private boolean checkIsInternalLoad(String user, String passwd, String db, String tbl,
+                                        String clientIp) {
+        for (Frontend fe : GlobalStateMgr.getCurrentState().getNodeMgr().getAllFrontends()) {
+            if (fe.getHost().equals(clientIp) && fe.isAlive() && fe.getHost().equals(user) &&
+                    fe.getNodeName().equals(passwd) && StatsConstants.STATISTICS_DB_NAME.equals(db) &&
```

Comment:
is fe node node default password?

Reply:
yes, sr's stream load need account/password, I don't have any good ideas